packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -4147,6 +4147,107 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/approve:
+    post:
+      tags:
+        - eservices
+      summary: approve a delegated new e-service version
+      operationId: approveDelegatedEServiceDescriptor
+      description: Approves a delegated E-Service descriptor
```

Consistency with the rest of the M2M spec
```suggestion
      summary: Approve a delegated E-Service Descriptor in Waiting for Approval state
      operationId: approveDelegatedEServiceDescriptor
      description: Approve a delegated E-Service Descriptor that is in Waiting for Approval state
```